dev_guide/builds.adoc

```diff
@@ -519,9 +531,42 @@ For example, defining a custom HTTP proxy to be used during build:
 ----
 ====
 
+[[source-code]]
+
+== Git Repository Source Options
+
+The source code location is one of the required parameters for the
```

Comment:
The only way to have Dockerfile as the only source is to have `bc.spec.strategy.type` equals to `Docker` or `Custom`, right?